### ISSUE TITLE
Empty technology pool when disabling knowledge from "the faction's tech level"

### ISSUE THREAD

**Issue by DarkShadow44** [jptrrs/HumanResources#181]
I tried to remove the setting "Knowledge Generation draws from... the faction's tech level" and all pawns spawn without any tech at all.
I'd expect them to know the starting techs like "Electricity" and similar.

To test just use this mod and create a new colony.

[Log.txt](https://github.com/jptrrs/HumanResources/files/9273861/Log.txt)

**Comment by jptrrs** [comment]
So, what did you set the knowledge generation to draw from?

**Comment by DarkShadow44** [comment]
Just from the faction's default starting techs and the scenario starting research. Since I use the default scenario, I should have electricity as starting tech, right?

**Comment by jptrrs** [comment]
Have you tried resetting the settings to default?

**Comment by DarkShadow44** [comment]
Yes, doesn't change anything. Should be easy to reproduce, or does it work for you?

**Comment by jptrrs** [comment]
After resetting to default,  you'll have this:
![image](https://user-images.githubusercontent.com/47060670/183261056-975c3df5-9d6d-4b05-8a18-eb87fdd4361f.png)
This will work just fine.
But if you remove "...the faction's tech level" and start a game on the default "crashlanded" scenario, then you'll start getting  empty pawns. BTW, you can see there's something wrong on the log bc this warning will pop up when any pawn is spawned: "[HumanResources] Empty technology pool!"
It should happen if, from the available starting techs, the game can't find any one matching the pawn's highest skills. For this reason, it's never a good idea to narrow the available techs too much.  (You can get more feedback on that process on the log if you turn on "Development mode" and "Verbose logging". The mod will detail knowledge generation for each pawn.)
The thing is, for the default scenario, The faction techs are: complex furniture, passive cooler, stonecutting, complex clothing, electricity, nutrient paste, air conditioning. For these, pawns with the max skill on construction, crafting, cooking and mining should have no trouble getting initial techs. But I can confirm: they are coming up empty, nonetheless. 
I'm going to have to take a look on what's going on. In the meantime, I advise leaving those settings on the default configuration.
